src/OrganisationRegistry.Api/ScheduledCommands/SyncFromKboService.cs

```diff
@@ -0,0 +1,79 @@
+﻿namespace OrganisationRegistry.Api.ScheduledCommands;
```

Comment:
er is veel overlap tussen de 2 hostedservices die we hebben, ik ben geneigd om hier een soort van HostedServiceBase ofzo van te maken ?